### PR TITLE
Remove Travis badge from the UI and API projects and investigate new badge

### PR DESCRIPTION
**Pull request by michho8**
# Ticket Link

[Ticket 1387](https://uhawaii.atlassian.net/browse/GROUPINGS-1387)

# List of squashed commits

- Delete Travis occurrences
- Add Test badge using push_pull_testing.yml
- Some suggested changes by IntelliJ and spelling
- Add back Groupings related lines in custom.properties
- Add back webService.login line
- Revert GroupAttributeTest.java

# Test Checklist

- [X] Unit Tests Passed:
- [X] Integration Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

**Comment by michho8** [comment]
Errors in application-integrationTest.properties, working with Kailee to resolve them.

**Comment by michho8** [comment]
Concluded that the errors are only on my side, probably a configuration problem that I need to work out later.

File: src/test/java/edu/hawaii/its/api/wrapper/GroupAttributeTest.java
Comment (michho8) [inline src/test/java/edu/hawaii/its/api/wrapper/GroupAttributeTest.java:1]: I can revert this, the IntelliJ suggestions may only be on my side as well.

**Review by mhodgesatuh — APPROVED** [review]
Thanks for catching the spelling error.  IntelliJ is good about pointing them out I've discovered.

**Review by JorWo — APPROVED** [review]

**Pull request merged**